proposed/2022/vulnerabilities-in-restore.md

```diff
@@ -80,6 +80,14 @@ This feature will be opt-in to start and gather feedback from developers.
 
 To enable the feature, a developer can add `<NuGetAudit>enable</NuGetAudit>` to their project file as a MSBuild property. To disable the feature, a developer can add `<NuGetAudit>disable</NuGetAudit>` or remove the property from the project file.
 
+#### Setting Vulnerability Auditing Modes
+
+There will be different modes to audit vulnerabilities based on the developer's or developer's team preference. To do this, a developer will opt-in to a feature called `<NuGetAuditMode>` which will have different modes such as `direct`, `transitive`, and `all`.

```

Comment:
When do we start discussing how these modes might be brought into the PM UI ? My first impressions are

- Actions (install/update) would audit automatically based on a user setting for AuditMode.
- Allow filtering package lists on AuditMode so you can easily see out of hundreds of packages: are my top-level packages vulnerable? Then, which transitives are vulnerable?

Happy to brainstorm more details on each bullet & other ideas.

Reply:
I'm going to schedule a larger team meeting to discuss this. It has come to my attention that we could make many experiences audit aware and show appropriately.